scenes/game/local_game.gd

```diff
@@ -4434,6 +4547,91 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 				# Didn't have any that matched, so forced to reveal hand.
 				_append_log_full(Enums.LogType.LogType_Effect, opposing_player, "has no matching cards so their hand is revealed.")
 				events += opposing_player.reveal_hand()
+		"remove_buddy_near_opponent":
+			ignore_extra_effects = true
+			var buddies = []
+			var same_space_allowed = effect['same_space_allowed']
+			var offset_allowed = effect['offset_allowed']
+			var optional = 'optional' in effect and effect['optional']
+			if same_space_allowed:
+				buddies += performing_player.get_buddies_on_opponent()
+			if offset_allowed == 1:
+				buddies += performing_player.get_buddies_adjacent_opponent()
+
+			if buddies.size() > 0:
+				decision_info.clear()
+				decision_info.type = Enums.DecisionType.DecisionType_ChooseArenaLocationForEffect
+				decision_info.player = performing_player.my_id
+				decision_info.choice_card_id = card_id
+				decision_info.effect_type = "remove_buddy_near_opponent"
+				decision_info.choice = []
+				decision_info.limitation = []
+				decision_info.source = effect['buddy_name']
+				var and_effect = null
+				var and_with_bonus = null
+				if 'and' in effect:
+					and_effect = effect['and']
+					and_with_bonus = and_effect
+				if 'if_removed_effect' in effect:
+					var additional_and_effect = and_effect
+					and_with_bonus = effect['if_removed_effect']
+					and_with_bonus['and'] = additional_and_effect
```

Comment:
would 'and' being set to null in an effect cause any issues? might be better to only set this if the and effect is non-null